### PR TITLE
Add React Slack Clone

### PR DESCRIPTION
**Pull request by bookercodes**
GitHub repo link: https://github.com/pusher/react-slack-clone

### CODE REVIEW

File: _data/projects/ReactSlackClone.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: React Slack Clone
+desc: Slack clone built with React and Chatkit
+site: https://pusher.github.io/react-slack-clone/
+tags:
+# Note those are tags categorizing your project, not issue labels.
+- JavaScript
+- js
+- web
+upforgrabs:
+  name: Help Wanted
```
Comment (shiftkey) [inline _data/projects/ReactSlackClone.yml]: Minor thing: the label is `help wanted` (lower case) in the issue tracker

**Comment by bookercodes** [comment]
@shiftkey Great catch -- fixed !

**Comment by ritwik12** [comment]
@bookercodes Thanks !!

**Comment by bookercodes** [comment]
Nice one mate, thanks as well!

**Pull request merged**